tests/Knp/DoctrineBehaviors/ORM/ActivableTest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace Tests\Knp\DoctrineBehaviors\ORM;
+
+use Knp\DoctrineBehaviors\Reflection\ClassAnalyzer;
+use Doctrine\Common\EventManager;
+
+require_once 'EntityManagerProvider.php';
+
+class ActivableTest
```

Comment:
It should extend ```\PHPUnit_Framework_TestCase```
Otherwise this test won't be detected by PHPUnit.
And yes, all the tests are failing. The full stacktrace you can check in the [gist](https://gist.github.com/akovalyov/e3c8b9590e334d1bc1d4) if you are interested